examples/plotting/histogram.py

```diff
@@ -16,16 +16,16 @@
 def make_plot(title, hist, edges, x, pdf, cdf):
     p = figure(title=title, tools='', background_fill_color="#fafafa")
     p.quad(top=hist, bottom=0, left=edges[:-1], right=edges[1:],
-           fill_color="navy", line_color="white", alpha=0.5)
-    p.line(x, pdf, line_color="#ff8888", line_width=4, alpha=0.7, legend_label="PDF")
-    p.line(x, cdf, line_color="orange", line_width=2, alpha=0.7, legend_label="CDF")
+           fill_color="#004488", line_color="#ffffff", alpha=0.5)
+    p.line(x, pdf, line_color="#BB5566", line_width=4, alpha=0.7, legend_label="PDF")
+    p.line(x, cdf, line_color="#DDAA33", line_width=4, alpha=0.7, legend_label="CDF")
 
     p.y_range.start = 0
     p.legend.location = "center_right"
     p.legend.background_fill_color = "#fefefe"
     p.xaxis.axis_label = 'x'
     p.yaxis.axis_label = 'Pr(x)'
-    p.grid.grid_line_color="white"
+    p.grid.grid_line_color="#ffffff"
```

Comment:
I think using named SVG color "white" is acceptable rather than a hex code (here and also above)